src/styles/colors.js

```diff
@@ -1,21 +1,31 @@
 export default {
-    white: '#FFFFFF',
-    gray1: '#FAFAFA',
-    gray2: '#ECECEC',
-    gray3: '#C6C9CA',
-    gray4: '#7D8B8F',
     dark: '#0b1b34',
     black: '#000000',
-    blue: '#0185ff',
-    blueHover: '#0063bf',
+    blue: '#5AB0FF',
```

Comment:
I don't think we actually want to change this value since this is our true brand blue. If we change it, we get something like this:
<img width="397" alt="image" src="https://user-images.githubusercontent.com/2319350/200697833-bd61d566-c0d1-423a-ac42-e9ed02438e77.png">


So we should still add the right blue that corresponds to our link blue, but do this while leaving blue as it is. 

So maybe it's a matter of naming the new one blue300 (since that is the new link color) and then the hover below is simply blue200 (since that is the new link hover color)